pkg/util/filesystem/open_windows.go

```diff
@@ -6,19 +6,19 @@
 //go:build windows
 // +build windows
 
-package file
+package filesystem
 
 import (
 	"os"
 
 	"golang.org/x/sys/windows"
 )
 
-// openFile reimplements the os.Open function for Windows because the default
+// OpenShared reimplements the os.Open function for Windows because the default
 // implementation opens files without the FILE_SHARE_DELETE flag.
 // cf: https://github.com/golang/go/blob/release-branch.go1.11/src/syscall/syscall_windows.go#L271
-// This prevents users from moving/removing files when the tailer is reading the file.
-func openFile(path string) (*os.File, error) {
+// This prevents users from moving/removing files when a log tailer is reading the file.
```

Comment:
This comment confused me at first, if I am understanding `FILE_SHARE_DELETE` correctly, it is needed to _allow_ other users to rename/remove files.

So when this comment says "This prevents users from ...", it is referring to the default behavior of go's `open` call, not the `OpenShared` version in this source file.

```suggestion
// Without FILE_SHARE_DELETE, other users cannot rename/remove the file while this handle is open. Adding this flag allows the agent to have the file open, while not preventing it from being rotated/deleted.
```

Reply:
That's a good point, thanks!  I copied the comment, but I'll update.